frontend/src/lib/components/MilestoneButton.svelte

```diff
@@ -20,16 +21,27 @@ let {
 } = $props();
 
 let bg_color = `bg-milestone-answer-${index}`;
-const text_color = index < 2 ? "dark:text-black" : "dark:text-white";
+const text_color =
+	index > 2 || index === -1
+		? "text-white dark:text-black"
+		: "text-gray dark:text-white";
 </script>
 
-<button
-	type="button"
-	onclick={onClick}
-	class={`${bg_color} ${text_color} ${selected ? 'opacity-100 outline-none ring-4 ring-blue-400' : 'opacity-75 hover:opacity-100'} border-1 m-1 rounded-lg border border-gray-200 px-5 py-3 text-center font-medium md:my- `}
->
-	{@render children?.()}
-</button>
-<Tooltip class={`${bg_color} text-gray-700 dark:${bg_color} dark:text-gray-700`}>
-	{tooltip}
-</Tooltip>
+<div class="flex flex-col">
+	<button
+			type="button"
+			onclick={onClick}
+			class={`${bg_color} ${text_color} ${selected ? 'opacity-100 outline-none ring-4 ring-blue-400' : 'opacity-75 hover:opacity-100'} border-1 m-1 rounded-lg border border-gray-200 px-5 py-3 text-center font-medium md:my- `}
+	>
+		{@render children?.()}
+	</button>
+
+	{#if tooltip}
+		<!-- Desktop: Use Tooltip component -->
+		<div class="hidden md:block">
+			<Tooltip class={`${bg_color} ${text_color} dark:${bg_color} `}>
+				{tooltip}
+			</Tooltip>
+		</div>
```

Comment:
for me the tooltip also disappeared on desktop - but it seems to work if `hidden md:block` is put inside the Tooltip class instead:
```suggestion
		<Tooltip class={`${bg_color} ${text_color} dark:${bg_color} hidden md:block`}>
			{tooltip}
		</Tooltip>
```